modules/metastore/config/install/workflows.workflow.dkan_publishing.yml

```
@@ -28,6 +28,11 @@ type_settings:
       default_revision: true
       label: Orphaned
       weight: 2
+    hidden:
+      published: true
+      default_revision: true
+      label: 'Publish without search indexing'
```

```suggestion
      label: 'Published (hidden)'
```
More consistent state labels